### PR TITLE
Second Set of Design Review Styling Changes

### PR DESCRIPTION
**Pull request by asilver-d2l**
Addresses issues:
 - Immersive navbar: There should be 30px between hamburger menu and edge of page
 - Immersive navbar: There should be 24px between the separator and the "back" control

Also added "is-sidebar" to help with functionality in d2l-sequence-navigator.

### CODE REVIEW

File: d2l-sequence-viewer.js [hunk 1]
```diff
@@ -143,7 +143,7 @@ class D2LSequenceViewer extends mixinBehaviors([
 		<div id="sidebar-occlude">
 		</div>
 		<div id="sidebar" class="offscreen">
-			<d2l-sequence-navigator href="{{href}}" token="[[token]]" role="navigation">
+			<d2l-sequence-navigator href="{{href}}" token="[[token]]" role="navigation" is-sidebar="true">
```
Comment (jwong-d2l) [inline d2l-sequence-viewer.js]: I think role="navigation" is what was used previously to determine if it was the sidebar or not.

Comment (asilver-d2l) [reply d2l-sequence-viewer.js]: There doesn't seem to be a `role` property that d2l-sequence-navigator has. Also, I'm curious why there is a `isSidebar` property in d2l-sequence-navigator if it is not used?

Comment (jwong-d2l) [reply d2l-sequence-viewer.js]: I think isSidebar used to read from the role property.  you can link them up again, or get rid of the role property.

Comment (jwong-d2l) [reply d2l-sequence-viewer.js]: I do think the role notation does seem a bit cleaner to understand though.

**Pull request merged**